File: aula6/exercicio1.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace exercicioscsharp {
	internal class Program {
		static void Main(string[] args) {
			Console.WriteLine("Digite a quantidade de numeros");
			int quantidadenumeros = int.Parse(Console.ReadLine());
			List<int> numeros = new List<int>();
			double soma = 0;

			for (int i = 0; i < quantidadenumeros; i++){
				Console.WriteLine("Insira um numero");
				int numero = int.Parse(Console.ReadLine());
				if (numero % 2 != 0) {
					Console.WriteLine("Numero impar ignorado");
					continue;
				}
				Console.WriteLine(numero);
				numeros.Add(numero);
				soma += numero;
			}
			Console.WriteLine("A media dos valores é: {0:F1}", soma / numeros.Count);
			Console.ReadKey();
		}
	}
}

File: aula8/exercicio1.cpp
#include <iostream>

int main() {
	int numeros[10] = {};
	for (int i = 0; i < 10; ++i) {
		std::cout << "digite um numero\n";
		std::cin >> numeros[i];
	}

	int pares[10] = {};
	int impares[10] = {};
	for (int i = 0; i < 10; ++i) {
		if (numeros[i] % 2 == 0) {
			pares[i] = numeros[i];
		}
		else {
			impares[i] = numeros[i];
		}
	}

	std::cout << "pares: ";
	for (int i = 0; i < 10; ++i) {
		if (pares[i] != 0) {
			std::cout << pares[i] << " ";
		}
	}

	std::cout << "impares: ";
	for (int i = 0; i < 10; ++i) {
		if (impares[i] != 0) {
			std::cout << impares[i] << " ";
		}
	}
}
